org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -32,6 +34,47 @@ public class MRTKRayInteractor :
     {
         #region MRTKRayInteractor
 
+        /// <summary>
+        /// Holds a reference to the <see cref="TrackedPoseDriver"/> associated to this interactor if it exists.
+        /// </summary>
+        private TrackedPoseDriver trackedPoseDriver = null;
+
+        /// <summary>
+        /// Property for accessing trackedPoseDriver which holds a reference to the <see cref="TrackedPoseDriver"/> associated to this interactor if it exists.
+        /// </summary>
+        private TrackedPoseDriver TrackedPoseDriver
+        {
+            get
+            {
+                if (trackedPoseDriver == null) //Try to get the TrackedPoseDriver component from the parent if it hasn't been set yet
+                {
+                    trackedPoseDriver = GetComponentInParent<TrackedPoseDriver>();
```

Comment:
I don't like the pattern of always calling GetComponentInParent in a getter.   If `GetComponentInParent` returns null and getter is called numerous times, we will have a performance hit.

If we want `GetComponentInParent`, do in Start

Reply:
done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/750/commits/24a6b203f2ef37d5ba7160a6daea0016d961b895 resolving